tekton/ci-workspace/eventlistener.yaml

```diff
@@ -0,0 +1,48 @@
+---
+apiVersion: triggers.tekton.dev/v1alpha1
+kind: EventListener
+metadata:
+  name: tekton-ci
+spec:
+  serviceAccountName: tekton-ci-listener
+  serviceType: NodePort
+  triggers:
+    - name: github-all-pull-request-ci
+      interceptors:
+        - github:
+            secretRef:
+              secretName: ci-webhook
+              secretKey: secret
+            eventTypes:
+              - pull_request
+        - cel:
+            filter: >-
+              body.repository.full_name.startsWith('afrittoli/') &&
+              body.repository.name in ['tektoncd-community', 'plumbing', 'pipeline', 'triggers', 'cli', 'dashboard', 'catalog', 'hub'] &&
+              (body.action == 'opened' ||
+               body.action == 'synchronize' ||
+               body.action == 'labeled' ||
+               body.action == 'unlabeled' ||
+               body.action == 'reopened')
+            overlays:
+              - key: git_clone_depth
+                expression: "string(body.pull_request.commits + 1.0)"
+        #     overlays:
```

Comment:
nit: suggest dropping commented out code (or leaving a comment explaining when it should be reinstated?)

Reply:
will do